packages/atomic/src/components/atomic-smart-snippet/atomic-smart-snippet-suggestions.tsx

```diff
@@ -0,0 +1,258 @@
+  shadow: true,
+})
+export class AtomicSmartSnippetSuggestions implements InitializableComponent {
+  @InitializeBindings() public bindings!: Bindings;
+  public smartSnippetQuestionsList!: SmartSnippetQuestionsList;
+  @BindStateToController('smartSnippetQuestionsList')
+  @State()
+  public smartSnippetQuestionsListState!: SmartSnippetQuestionsListState;
+  public error!: Error;
+  @Element() public host!: HTMLElement;
+
+  /**
+   * The [heading level](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) to use for the "People also ask" heading over the snippets, from 1 to 5.
+   *
+   * We recommend setting this property in order to improve accessibility.
+   *
+   */
+  @Prop({reflect: true}) public headingLevel = 0;
+
+  /**
+   * Sets the style of the snippets.
+   *
+   * Example:
+   * ```ts
+   * smartSnippet.snippetStyle = `
+   *   b {
+   *     color: blue;
+   *   }
+   * `;
+   * ```
+   */
+  @Prop({reflect: true}) snippetStyle?: string;
+
+  private id = randomID('atomic-smart-snippet-suggestions-');
+
+  public initialize() {
+    this.smartSnippetQuestionsList = buildSmartSnippetQuestionsList(
+      this.bindings.engine
+    );
+  }
+
+  private getRelatedQuestionId(relatedQuestion: SmartSnippetRelatedQuestion) {
+    // TODO: Change to the snippet's unique ID
+    return `${this.id}-${relatedQuestion.question
+      .toLowerCase()
+      .replace(/\s/g, '-')
+      .replace(/[^a-z-]/g, '')}`;
+  }
+
+  private get style() {
+    const styleTag = this.host
+      .querySelector('template')
+      ?.content.querySelector('style');
+    if (!styleTag) {
+      return this.snippetStyle;
+    }
+    return styleTag.innerHTML;
+  }
+
+  private renderQuestion(relatedQuestion: SmartSnippetRelatedQuestion) {
+    const toggleRelatedQuestion = () =>
+      relatedQuestion.expanded
+        ? this.smartSnippetQuestionsList.collapse(relatedQuestion.documentId)
+        : this.smartSnippetQuestionsList.expand(relatedQuestion.documentId);
+
+    return (
+      <Button
+        style="text-neutral"
+        part={
+          relatedQuestion.expanded
+            ? 'question-expanded-button'
+            : 'question-collapsed-button'
+        }
+        onClick={toggleRelatedQuestion}
+        class="flex items-center px-4"
+        ariaExpanded={`${relatedQuestion.expanded}`}
+        ariaControls={this.getRelatedQuestionId(relatedQuestion)}
+      >
+        <atomic-icon
+          icon={relatedQuestion.expanded ? ArrowDown : ArrowRight}
+          part={
+            relatedQuestion.expanded
+              ? 'question-expanded-icon'
+              : 'question-collapsed-icon'
+          }
+          class="w-2.5 mr-3 stroke-[1.25]"
+        ></atomic-icon>
+        <Heading
+          level={this.headingLevel ? this.headingLevel + 1 : 0}
+          class="text-left text-xl font-bold py-4"
+          part={
+            relatedQuestion.expanded
+              ? 'question-expanded-text'
```

Comment:
especially if you follow my suggestion, I'd create a `makePart` function to do this, like `this.makePart('question-text')`

Reply:
or - may be overkill, but.... - since you have so many ternaries with the expanded, something like `this.ifExpanded('question-text expanded', 'question-text collapsed')`, 
```
    const toggleRelatedQuestion = () =>
      this.ifExpanded(
        this.smartSnippetQuestionsList.collapse(relatedQuestion.documentId),
        this.smartSnippetQuestionsList.expand(relatedQuestion.documentId)
      )
```

Reply:
Sure! I'll add `this.suffixQuestionPart(relatedQuestion, 'question-text')`

Reply:
I'll extract a method to toggle a related question, like `this.toggleRelatedQuestion(relatedQuestion)`